src/cache/lru_cache_manager/lru_queue.h

```diff
@@ -21,15 +22,16 @@ typedef struct LRUQueue {
   LRUNode *tail;            // Queue tail
   LRUNode *emptySpace;      // Next empty place in the queue
```

Comment:
rename to freeList, shouldn't that be an array of indices into queue?

Reply:
there is (single) next empty space and a free list